packages/next/src/utils.ts

```diff
@@ -2579,6 +2580,9 @@ export async function getMiddlewareBundle({
         };
 
         route.middlewarePath = shortPath;
+        route.middlewareRawSrc = matcher.originalSource
+          ? [matcher.originalSource]
```

Comment:
Do we want to fallback to the `regex` when `originalSource` isn't available or is this done upstream already?

Reply:
No need, the front will automatically fallback: https://github.com/vercel/front/pull/19606.